classes/migration/upgrade/v3_4_0/PreflightCheckMigration.php

```diff
@@ -90,9 +90,23 @@ protected function buildOrphanedEntityProcessor(): void
             }
             $affectedRows += $this->deleteOptionalReference('publications', 'section_id', 'sections', 'section_id');
             // Remaining cleanups are inherited
+            $rows = DB::table('publications AS p')
```

Comment:
The same as above, you don't need to touch the upgrade scripts for versions < 3.5.